modules/configuring-assumerole-without-iam.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="configuring-assumerole-without-iam_{context}"]
+= Configuring AssumeRole without container IAM
+
+[role="_abstract"]
+If you want to use AssumeRole without container IAM, you must use an an access and a secret key to authenticate as an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[AWS user with programmatic access].
```

Comment:
Typo `use an an access`, more cleanly written as follows:
`you must use an access key and a secret key...`
~or~
'you must use an access and secret key`